scheduling/priority_queue.py

```diff
@@ -0,0 +1,155 @@
+# Implementation of Priority Queue scheduling algorithm (without preemption)
+# In this Algorithm we only care about the order that the processes arrived
+# without caring about their duration time
+# https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling
+from __future__ import annotations
+
+
+def sort_processes_by_priority(
+    processes: list[int], duration_times: list[int], priorities: list[int]
+) -> list[int]:
```

Comment:
```suggestion
    processes: List[int], duration_times: List[int], priorities: List[int]
) -> List[int]:
```

Reply:
It was due to the doctests in a function.